packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-glossary.tsx

```diff
@@ -112,6 +120,61 @@ const ContractTypeGlossary = ({ category }: { category: string }) => {
                     },
                 ];
                 break;
+            case 'multiplier':
+                content = [
+                    { type: 'heading', text: localize('Stop out') },
+                    {
+                        type: 'paragraph',
+                        text: localize(
+                            'Your trade will be closed automatically at the nearest available asset price when your loss reaches a certain percentage of your stake, but your loss never exceeds your stake. This percentage depends on the chosen underlying asset and the Multiplier.'
+                        ),
+                    },
+                    { type: 'heading', text: localize('Take profit') },
+                    {
+                        type: 'paragraph',
+                        text: localize(
+                            'If you select this feature, your trade will be closed automatically at the nearest available asset price when your profit reaches or exceeds the take profit amount. Your profit may be more than the amount you entered depending on the market price at closing.'
+                        ),
+                    },
+                    { type: 'heading', text: localize('Stop loss') },
+                    {
+                        type: 'paragraph',
+                        text: localize(
+                            'If you select this feature, your trade will be closed automatically at the nearest available asset price when your loss reaches or exceeds the stop loss amount. Your loss may be more than the amount you entered depending on the market price at closing.'
+                        ),
+                    },
+                    { type: is_multiplier_fx ? '' : 'heading', text: localize('Deal cancellation') },
+                    {
+                        type: is_multiplier_fx ? '' : 'paragraph',
+                        text: localize(
+                            'If you select this feature, you can cancel your trade within a chosen time frame if the asset price moves against your favour. You will get your stake back without profit/loss. We charge a small fee for this. Take profit and stop loss are disabled when deal cancellation is active.'
+                        ),
+                    },
+                    { type: 'heading', text: localize('Slippage risk') },
+                    {
+                        type: 'paragraph',
+                        text: localize(
+                            'Slippage happens when the asset price changes by the time it reaches our servers.'
+                        ),
+                    },
+                    { type: 'heading', text: localize('Entry spot') },
+                    {
+                        type: 'paragraph',
+                        text: is_major_pairs
```

Comment:
Calculations for `Major Pairs` is different and based on next-tick-excution mechanism, so for such symbols we should show different content.